index.html

```diff
@@ -1430,10 +1430,16 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
       Promise&lt;void&gt; push(NFCMessageSource message, optional NFCPushOptions options);
     };
 
-    [Constructor(), SecureContext, Exposed=Window]
+    [Constructor(optional NFCReaderOptions options), SecureContext, Exposed=Window]
     interface NFCReader : EventTarget {
-      Promise&lt;void&gt; watch(optional NFCWatchOptions options);
+      readonly attribute USVString url;
+      readonly attribute NFCRecordType recordType;
+      readonly attribute USVString mediaType;
+      readonly attribute NFCAccept accept;
```

Comment:
Just curious, what is the motivation for exposing these to read out? It seems like this will just be the options that were passed in, so I don't really see the use case. Did you just add these attributes so that they were stored internally in the NFCReader? You  might want to use internal slots for that purpose, if so.